protocol/examples/messageData/vote_cast_write_in.json

```diff
@@ -0,0 +1,14 @@
+{
+    "object": "election",
+    "action": "cast_vote",
+    "lao": "XXX",
+    "election": "XXX",
+    "created_at": 123,
+    "votes": [
+        {
+            "id": "XXX",
+            "question": "XXX",
+            "write_in": "XXX"
```

Comment:
is this really supported? 🤔 I don't think so, to be honest.
It may be best to remove this for now (incl. from the documentation and the application, as needed)